src/mmw/js/src/modeling/models.js

```diff
@@ -10,6 +10,24 @@ var Backbone = require('../../shim/backbone'),
     turfErase = require('turf-erase'),
     turfIntersect = require('turf-intersect');
 
+var GWLFE = 'gwlfe';
+var TR55_TASK = 'tr55';
+var TR55_PACKAGE = 'tr-55';
+
+var modelPackages = [
+    {
+        name: TR55_PACKAGE,
+        displayName: 'Site Storm Model',
+        description: ''
+    },
+    {
+        name: GWLFE,
+        displayName: 'Watershed Multi-Year Model',
+        description: '',
+        disabled: true
```

Comment:
True, but if this is acting as a feature flag, which we turn on for development but off for staging / production deploys, it makes sense to keep it with all other feature flag-like settings, so there's one place for everything.